news/2012-03-06-webinar-outside-in.html

```diff
@@ -0,0 +1,137 @@
+                </span>
+            </div>
+
+            <!-- This is the div that will contain the Preference Editor component -->
+            <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the desktop view.
+                A separate panel bar for desktop displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-wideScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+        </div>
+
+        <div class="floe">
+            <header class="floe-header">
+                <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">floe
+                    <span class="floe-header-tagline">
+                        flexible learning for open education
+                    </span></a>
+                </h1>
+            </header>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <article class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <h3>Webinar: Outside In - Achieving Our Goals by Designing for Diversity</h3>
+                <time class="floe-date" datetime="2012-03-06">March 06, 2012</time>
+                <p>
+                    Jutta Treviranus, Director of the Inclusive Design Research Centre at OCAD University, will be co-presenting a webinar as part of
+                    <a href="http://www.openeducationweek.org/">Open Education Week</a>
+                    . The aim of Open Education Week is to raise awareness of the open education movement and its impact on teaching and learning worldwide. The webinar, entitled "Outside In - Achieving Our Goals by Designing for Diversity," will discuss how processes that design open courseware resources and delivery platforms for diverse learners also address many other goals, including deeper learning, delivery on mobile platforms, discovery, internationalization, and many more.
+                </p>
+                <p>
+                    The webinar will be presented Thursday, March 8, 2012 at 22:00 GMT.
+                </p>
+                <p>
+                    For more information about the webinar, visit
+                    <a href="http://www.openeducationweek.org/webinars/">the webinar page on the Open Education Week website</a>
```

Comment:
This link leads to an error page and there are no replacement links available for the webinars section of the website. I propose to just remove this link altogether and leave the [http://www.openeducationweek.org/](url) up instead.